lib/marc_indexer.rb

```diff
@@ -198,7 +198,9 @@
 to_field 'title_main_first_char_ssim', extract_title_main_first_char
 to_field 'title_precise_tesim', extract_title_precise
 to_field 'title_series_ssim', extract_marc(title_series_str(ATOG))
+to_field 'title_series_ssim', extract_8xx_with_subfield3(ATOG)
```

Comment:
@alexBLR Is it an issue that the fields are being extracted to twice, or is that the intended behavior?

Reply:
Discussed my concerns via slack.